### ISSUE TITLE
Bags show no items

### ISSUE THREAD

**Issue by Dannilion** [Cidan/BetterBags#482]
### Addon Version

v0.1.79

### World of Warcraft Version

Retail

### Describe the bug

When the bag is opened with the new patch, nothing shows inside.

### Steps to reproduce

Open bag with no other addons installed.
See nothing.
![betterbagsbroke](https://github.com/user-attachments/assets/13ff3a88-9211-4352-987a-35778ea7da88)

**Comment by AMaribo** [comment]
It seems like you have a Lua error. If you could post them here, that would help.
It might be related to this issue.
[Issue 481](https://github.com/Cidan/BetterBags/issues/481#issuecomment-2248668302)

**Comment by Dannilion** [comment]
26x .../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua:294: attempt to index local 'color' (a number value)
[string "@Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:294: in function `SetItemButtonOverlay'
[string "@Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:249: in function <.../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua:236>
[string "@Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:672: in function <.../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua:671>
[string "=[C]"]: in function `SetItemButtonQuality'
[string "@Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:265: in function `SetItemButtonQuality'
[string "@BetterBags/frames/item.lua"]:333: in function `SetItemFromData'
[string "@BetterBags/frames/item.lua"]:286: in function `SetItem'
[string "@BetterBags/views/gridview.lua"]:88: in function <BetterBags/views/gridview.lua:80>
[string "@BetterBags/views/gridview.lua"]:164: in function `Render'
[string "@BetterBags/frames/bag.lua"]:251: in function `Draw'
[string "@BetterBags/core/init.lua"]:257: in function `cb'
[string "@BetterBags/core/events.lua"]:50: in function <BetterBags/core/events.lua:45>
[string "=[C]"]: ?
[string "@PetTracker/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:29: in function <...ker/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:25>
[string "@PetTracker/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:64: in function `SendMessage'
[string "@BetterBags/core/events.lua"]:186: in function <BetterBags/core/events.lua:185>

Locals:
button = BetterBagsItemButton691DecorationElvUI {
 IconOverlay = Texture {
 }
 AugmentBorderAnim = AnimationGroup {
 }
 showMatchHighlight = true
 icon = BetterBagsItemButton691DecorationElvUIIconTexture {
 }
 IsSkinned = true
 count = 0
 HighlightTexture = Texture {
 }
 PixelSnapDisabled = true
 IconQuestTexture = BetterBagsItemButton691DecorationElvUIIconQuestTexture {
 }
 hasItem = 1
 timeSinceUpgradeCheck = 0
 PushedTexture = Texture {
 }
 flash = Texture {
 }
 isProfessionItem = false
 emptyBackgroundAtlas = "bags-item-slot64"
 Count = BetterBagsItemButton691DecorationElvUICount {
 }
 hover = Texture {
 }
 searchOverlay = BetterBagsItemButton691DecorationElvUISearchOverlay {
 }
 backdrop = Frame {
 }
 itemContextMatchResult = 3
 IconTexture = BetterBagsItemButton691DecorationElvUIIconTexture {
 }
 IconOverlay2 = Texture {
 }
 ItemSlotBackground = Texture {
 }
 minDisplayCount = 1
 flashAnim = AnimationGroup {
 }
 newitemglowAnim = AnimationGroup {
 }
 pushed = Texture {
 }
 JunkIcon = Texture {
 }
 ExtendedSlot = Texture {
 }
 BattlepayItemTexture = Texture {
 }
 NewItemTexture = Texture {
 }
 isCraftedItem = false
 AugmentBorderAnimTexture = Texture {
 }
 UpgradeIcon = Texture {
 }
 NormalTexture = BetterBagsItemButton691DecorationElvUINormalTexture {
 }
 IconBorder = Texture {
 }
 Cooldown = BetterBagsItemButton691DecorationElvUICooldown {
 }
 ItemContextOverlay = Texture {
 }
 BagIndicator = Texture {
 }
 Stock = BetterBagsItemButton691DecorationElvUIStock {
 }
}
itemIDOrLink = "|cffa335ee|Hitem:182138::::::::70:256::23:2:1579:7245:2:28:2044:37:9:::::|h[Mind Devourer]|h|r"
quality = 4
isBound = true
color = 0.639216
(*temporary) = Texture {
 PixelSnapDisabled = true
}
(*temporary) = 0.207843
(*temporary) = 0.933333
(*temporary) = "ffa335ee"
(*temporary) = "attempt to index local 'color' (a number value)"

**Comment by Cidan** [comment]
This is a bug in Blizzard's code -- this breaks default bags as well, due to Conduit items. Blizzard has to fix this.

**Comment by AMaribo** [comment]
It is the same issue as the one I linked.
`itemIDOrLink = "|cffa335ee|Hitem:182138::::::::70:256::23:2:1579:7245:2:28:2044:37:9:::::|h[Mind Devourer]|h|r"`
Turn off BetterBags, then delete this item [Mind Devourer](https://www.wowhead.com/item=182138/mind-devourer), then turn BetterBags back on.

**Comment by Dannilion** [comment]
Ah, thank you! Should I avoid doing Shadowlands content that might get me conduits until there's a fix then?

**Comment by supmiller** [comment]
Same thing happens to me unfortunately.

**Comment by AMaribo** [comment]
> Should I avoid doing Shadowlands content that might get me conduits until there's a fix then?

That is probably the safest option. Hopefully Blizzard will fix the issue soon.

**Comment by kcoderhere** [comment]
Same thing was happening to me but it eventually showed up.  Then I opened up my guild bank and everything was gone again.

```
Date: 2024-07-24 22:54:55
ID: 2
Error occured in: Global
Count: 1
Message: .../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua line 294:
   attempt to index local 'color' (a number value)
Debug:
   [string "@Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:294: SetItemButtonOverlay()
   [string "@Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:249:
      .../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua:236
   [string "@Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:672: SetItemButtonQuality()
   [string "@Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:265: SetItemButtonQuality()
   [string "@Interface/AddOns/BetterBags/frames/item.lua"]:333: SetItemFromData()
   [string "@Interface/AddOns/BetterBags/frames/item.lua"]:286: SetItem()
   [string "@Interface/AddOns/BetterBags/views/gridview.lua"]:88:
      Interface/AddOns/BetterBags/views/gridview.lua:80
   [string "@Interface/AddOns/BetterBags/views/gridview.lua"]:164: Render()
   [string "@Interface/AddOns/BetterBags/frames/bag.lua"]:258: Draw()
   [string "@Interface/AddOns/BetterBags/core/init.lua"]:257: cb()
   [string "@Interface/AddOns/BetterBags/core/events.lua"]:50:
      Interface/AddOns/BetterBags/core/events.lua:45
   [string "=[C]"]: ?
   [string "@Interface/AddOns/AllTheThings/lib/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:19:
      ...ings/lib/CallbackHandler-1.0/CallbackHandler-1.0.lua:15
   [string "@Interface/AddOns/AllTheThings/lib/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:54: SendMessage()
   [string "@Interface/AddOns/BetterBags/core/events.lua"]:186:
      Interface/AddOns/BetterBags/core/events.lua:185
Locals:
button = BetterBagsItemButton592Decorationdefault {
 IconOverlay = Texture {
 }
 AugmentBorderAnim = AnimationGroup {
 }
 showMatchHighlight = true
 icon = BetterBagsItemButton592DecorationdefaultIconTexture {
 }
 count = 0
 HighlightTexture = Texture {
 }
 IconQuestTexture = BetterBagsItemButton592DecorationdefaultIconQuestTexture {
 }
 hasItem = 1
 PushedTexture = Texture {
 }
 emptyBackgroundAtlas = "bags-item-slot64"
 timeSinceUpgradeCheck = 0
 Count = BetterBagsItemButton592DecorationdefaultCount {
 }
 flash = Texture {
 }
 searchOverlay = BetterBagsItemButton592DecorationdefaultSearchOverlay {
 }
 isProfessionItem = false
 itemContextMatchResult = 3
 IconTexture = BetterBagsItemButton592DecorationdefaultIconTexture {
 }
 IconOverlay2 = Texture {
 }
 ItemSlotBackground = Texture {
 }
 minDisplayCount = 1
 flashAnim = AnimationGroup {
 }
 newitemglowAnim = AnimationGroup {
 }
 JunkIcon = Texture {
 }
 ExtendedSlot = Texture {
 }
 BattlepayItemTexture = Texture {
 }
 NewItemTexture = Texture {
 }
 isCraftedItem = false
 AugmentBorderAnimTexture = Texture {
 }
 UpgradeIcon = Texture {
 }
 NormalTexture = BetterBagsItemButton592DecorationdefaultNormalTexture {
 }
 IconBorder = Texture {
 }
 Cooldown = BetterBagsItemButton592DecorationdefaultCooldown {
 }
 ItemContextOverlay = Texture {
 }
 BagIndicator = Texture {
 }
 Stock = BetterBagsItemButton592DecorationdefaultStock {
 }
}
itemIDOrLink = "[Condensed Anima Sphere]"
quality = 4
isBound = true
color = 0.639216
(*temporary) = Texture {
}
(*temporary) = 0.207843
(*temporary) = 0.933333
(*temporary) = "ffa335ee"
(*temporary) = "attempt to index local 'color' (a number value)"

```

**Comment by AMaribo** [comment]
It looks like you have [this](https://www.wowhead.com/item=187506/condensed-anima-sphere) generic conduit item in your guild bank.

**Comment by AMaribo** [comment]
Fixed in WoW version 11.0.0.55824.
